server/config/express.js

```diff
@@ -1,32 +1,30 @@
-var express = require('express');
-var passport = require('passport');
-var morgan = require('morgan');
-var bodyParser = require('body-parser');
-var cookieSession = require('cookie-session');
-var config = require('./config');
-var logger = require('./logger');
+let express = require('express');
+let passport = require('passport');
+let morgan = require('morgan');
+let bodyParser = require('body-parser');
+let cookieSession = require('cookie-session');
+let config = require('./config');
 
-var forceSsl = function (req, res, next) {
-  logger.log("req.headers['x-forwarded-proto'] = " + req.headers['x-forwarded-proto']);
-  if (config.env === 'production' && req.headers['x-forwarded-proto'] !== 'https') {
-    logger.log("req.headers['x-forwarded-proto'] = " + req.headers['x-forwarded-proto']);
-    return res.redirect(['https://', req.get('Host'), req.url].join(''));
-  }
-  return next();
+let forceSsl = function (req, res, next) {
```

Comment:
X-Forwarded-Proto isn't usually in requests from my experience. req.secure checks for SSL and is already in express. Seemed like a better option to me unless there is some historic reason for x-forwarded-proto